code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -173,6 +173,12 @@
 	icon_state = "bottleofnothing"
 	list_reagents = list("nothing" = 100)
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/bottleofbanana
+	name = "Jolly Jug"
```

Comment:
If it makes you feel better, it is more or less being used as a proper noun, more so then bottle of nothing, and would be capitalized anyway.